test/integration/master/synthetic_master_test.go

```diff
@@ -300,30 +300,34 @@ func TestObjectSizeResponses(t *testing.T) {
 
 	client := clientsetv1.NewForConfigOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Groups[api.GroupName].GroupVersion()}})
 
-	const DeploymentMegabyteSize = 100000
-	const DeploymentTwoMegabyteSize = 1000000
+	const labelString = ""
+	const finalizerString = "sample/sample"
+	// for given labelString and finalizerString, size of the object returned by constructBody is increased by factor*(size passed)
+	const factor = 16
```

Comment:
Why is factor 16? Could you add documentation on that?

Reply:
For a label or annotation, each iteration adds a key of length 10 and a value of length length(labelString). I'm not sure why, but that has a total size of 16.
For a finalizer, each iteration appends finalizerString to a []string{}. Again not sure why, but for length(finalizerString)=13, that has a total size of 16.